src/lib/__tests__/server-utilities.test.js

```diff
@@ -38,16 +40,31 @@ describe('md5Hash', () => {
 
 describe('compressJPEG', () => {
   if (!fs.existsSync('./tmp')) fs.mkdirSync('./tmp');
-  fs.closeSync(fs.openSync('./tmp/test.jpg', 'a')); // Create empty file 'test.jpg'
   const filePath = './tmp/test.jpg';
 
-  it('overwrites uncompressed image with compressed image', () => {
+  // Copy compressible JPEG to `./tmp` directory to mimick JPEG upload
+  copyFile('./static/test-assets/compression-test.jpg', filePath);
```

Comment:
I wouldn't put this in static as that's what we serve to the client, this means that the gazelle url `thegazelle.org/test-assets/compression-test.jpg` would show up the image. I'd just put it in the tests folder it's currently in and maybe as you did here make an assets folder. So in absolute terms in `src/lib/__tests__/assets` maybe?